qubesbuilder/plugins/build_archlinux/scripts/update-local-repo.sh

```diff
@@ -49,8 +49,8 @@ chroot_cmd sed "s/^ *CheckSpace/#CheckSpace/g" -i /etc/pacman.conf
 
 # Update archlinux keyring first so that Archlinux can be updated even after a long time
 chroot_cmd /bin/sh -c \
-    "http_proxy='${REPO_PROXY}' pacman -Sy --noconfirm --noprogressbar archlinux-keyring"
+    "no_proxy='${NO_PROXY}' http_proxy='${REPO_PROXY}' pacman -Sy --noconfirm --noprogressbar archlinux-keyring"
```

Comment:
What this `no_proxy` variable is for, and where `NO_PROXY` is set? If `NO_PROXY` is set already, that should be good enough for pacman (at least when it uses curl) already, no?

Reply:
> why else is `REPO_PROXY` here at all? ;))

It's _almost_ set here: https://github.com/QubesOS/qubes-builderv2/blob/main/qubesbuilder/plugins/build_archlinux/__init__.py#L142

Reply:
But still, something needs to set `NO_PROXY`, no?

Reply:
I mean, why not set it by default to 127.0.0.1 like in the other place?